### DISCUSSION TITLE
Typora

### DISCUSSION THREAD

**Discussion by rea1shane** [catppuccin/catppuccin#1855] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
No

**Additional context 📷 (optional)**
https://typora.io/

**Comment by justTOBBI** [comment]
I'd be down

**Comment by justTOBBI** [comment]
Hey, due to my lack of time and Typora being paid after the 7-day trial, I will not continue to make the port. I didn't make a lot of progress, so I won't even bother sending them here, as I'm unsure it couldn't be done better. Sorry and I hope someone will create a port :)

**Comment by ucomru** [comment]
> Hey, due to my lack of time and Typora being paid after the 7-day trial, I will not continue to make the port. I didn't make a lot of progress, so I won't even bother sending them here, as I'm unsure it couldn't be done better. Sorry and I hope someone will create a port :)

Good day!
I can to buy Typora license key to you for the port... 
If it is to you useful to work.

**Reply by justTOBBI** [reply]
Thank you for the willingsness to buy a typora key for me! As I am entering my A Level exams soon, I won't be free to port typora. If anyone else is willing to do that, please feel free to!

**Comment by Cliffback** [comment]
I'm using Typora myself, so threw up a quick theme based on the onedark typora theme. It's not finished yet, but works fine for now.

[https://github.com/Cliffback/typora-catppuccin-theme](https://github.com/Cliffback/typora-catppuccin-theme)
